statsmodels/stats/proportion.py

```diff
@@ -123,6 +123,185 @@ def func(qi):
         raise NotImplementedError('method "%s" is not available' % method)
     return ci_low, ci_upp
 
+def multinomial_proportions_confint(counts, alpha=0.05, method='goodman'):
+    '''Confidence intervals for multinomial proportions.
+
+    Parameters
+    ----------
+    counts : array_like of int, 1-D
+        Number of observations in each category.
+    alpha : float in (0, 1), optional
+        Significance level, defaults to 0.05.
+    method : {'goodman', 'sison_glaz'}, optional
```

Comment:
My mistake in proportion_confint, or no convention, changing convention in the past.

compromise: use `sison-glaz' in the docstring
but only check whether method name starts with "sison".
In some cases, I also allowed short cuts or multiple names for the keywords, when there is no unique favorite for the exact spelling. (alternative='two-sided'  '2s' ... IIRC)